unit-tests/run-unit-tests.py

```diff
@@ -459,8 +461,12 @@ def test_wrapper( test, configuration = None, repetition = 1 ):
                 continue
             #
             if skip_live_tests:
-                log.w( test.name + ':', 'is live and there are no cameras; skipping' )
-                continue
+                if skip_disconnected:
+                    log.w( test.name + ':', 'is live and there are no cameras; skipping' )
```

Comment:
It's no longer the case that there are no cameras. We should say that we're skipping because `--skip-disconnected` was specified.
Also, is it still a warning?

Reply:
It is also the case, it's an inner condition.
There is no camera and the flag is on. I'll add it to the log

Reply:
`log.w( test.name + ':', 'is live and no cameras found and skip_disconnected is on; skipping' )`

Reply:
Done